### PR TITLE
Update groups tutorial for project-wide node groups

### PR DESCRIPTION
**Pull request by Dowsley**
https://github.com/godotengine/godot/pull/60965 introduced several changes to how we manage groups in the Editor UI. The Groups dock has had an overhaul and the Groups Editor was removed entirely.

As https://github.com/godotengine/godot-docs/issues/9177 points out, this change is yet to be reflected in the documentation. This PR has the intent to be the first step towards this.

Please let me know how I can improve the writing, or any further page that should also change.

### CODE REVIEW

File: tutorials/scripting/groups.rst [hunk 1]
```diff
@@ -21,51 +21,50 @@ removed by removing all nodes from a given group.
...
 
-1. A list of groups used by nodes in the current scene.
-2. A list of nodes that are not part of the selected group.
-3. A list of nodes in the group.
+All groups present in the project that were marked as Global, created from any scene, will be visible under Global Groups. Any other group derived from nodes in the current scene will appear under Scene Groups.
 
-The fields at the bottom allow you to add new groups or filter nodes in the
-second and third columns.
+.. warning:: The same underlying logic is used for both Global and Scene groups. 
+            Groups with the same name are considered one and the same. This feature is purely organizational.
```
Comment (AThousandShips) [inline tutorials/scripting/groups.rst:60]:
```suggestion
             Groups with the same name are considered one and the same. This feature is purely organizational.
```

File: tutorials/scripting/groups.rst [hunk 2]
```diff
@@ -21,51 +21,50 @@ removed by removing all nodes from a given group.
...
+Selected Node(s) from the Scene dock can be added into groups by marking the checkbox on the left side
+of the groups in the Groups dock. The node(s) you had selected when creating a new group will be automatically checked.
 
-It has three columns:
+.. image:: img/groups_node_tab_with_created_groups.webp
 
-1. A list of groups used by nodes in the current scene.
-2. A list of nodes that are not part of the selected group.
-3. A list of nodes in the group.
+All groups present in the project that were marked as Global, created from any scene, will be visible under Global Groups. Any other group derived from nodes in the current scene will appear under Scene Groups.
```
Comment (AThousandShips) [inline tutorials/scripting/groups.rst:57]:
```suggestion
All groups present in the project that were marked as Global, created from any scene, will be visible under Global Groups.
Any other group derived from nodes in the current scene will appear under Scene Groups.
```
Break up long lines

**Comment by Dowsley** [comment]
Fixed in a force-pushed amend. Is it recommended that I add a new commit for fix next time?

**Comment by AThousandShips** [comment]
No we prefer single commits!

**Review by AThousandShips — APPROVED** [review]
The style itself looks good, will await some approval on content as well

**Comment by Dowsley** [comment]
> The style itself looks good, will await some approval on content as well

Thanks. Yes there anybody I can tag here or should I wait for a contributor to pick it?

**Comment by AThousandShips** [comment]
Relevant team is already requested for review

**Review by skyace65 — APPROVED** [review]

**Comment by skyace65** [comment]
Thanks! And congrats on your first merged PR!

**Pull request merged**